src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -208,6 +208,11 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
         return estimatedFees
       } catch (e: unknown) {
         console.debug(e)
+
+        if (e instanceof ChainAdapterError) {
+          throw new Error(e.metadata.translation)
```

Comment:
we don't have any interpolation options for this error right now (or probably ever), but this future proofs it and also makes it clear it is a translation since it is a bit hard to track down and know that this error thrown is going to be set to a form value that is ultimately used for the button text which happens to be wrapped in `translate`.

```suggestion
          throw new Error(translate(e.metadata.translation, e.metadata.options))
```

Reply:
Nice catch 🧠 doesn't hurt to for the sake of correctness! 5a261310ca

<img width="504" alt="image" src="https://github.com/user-attachments/assets/a0f4e5df-a249-43db-b74f-b5ef46d60ebe" />